src/components/jobbonsker.tsx

```diff
@@ -60,14 +66,49 @@ const Jobbonsker = () => {
         );
     }
 
+    if (!underOppfolgingData?.underOppfolging) {
+        return (
+            <Panel border className="info_panel" tabIndex={5}>
+                <Heading spacing level="2" size="medium" className="PanelHeader">
```

Comment:
Bruke klassenavnkonvensjonen som Sanne har tatt i bruk i PR-en sin (gjelder flere steder i koden):

```suggestion
                <Heading spacing level="2" size="medium" className="panel_header">
```